provider/aws/formation/app.json.tmpl

```diff
@@ -213,6 +220,26 @@
   {{ end }}
 {{ end }}
 
+{{ define "domain-condition" }}
+  {{ range .Services }}
+    "{{ upper .Name }}DefaultDomain": { "Fn::Equals": [ { "Ref": "{{ upper .Name }}DefaultDomain" }, "Yes" ] },
```

Comment:
Let's use just one `DefaultDomain` parameter to avoid the parameter proliferation (CloudFormation has a hard limit of 60 parameters per stack)

Reply:
Okay, I will update it with one parameter.